src/string_cleanup.py

```diff
@@ -0,0 +1,78 @@
+import re
+
+
+class StringCleanup:
```

Comment:
to follow the python stardards, it s good that all the classes that are not inheriting from any other, actually inherits from `object`